README.md

```diff
@@ -13,153 +13,112 @@ SPDX-License-Identifier: Apache-2.0
     <a href="https://pkg.go.dev/mod/github.com/shipwright-io/build"> <img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white"></a>
 </p>
 
-# Shipwright - a framework for building container images on Kubernetes
+# Shipwright
 
-Shipwright is an extensible framework for building container images on Kubernetes. With Shipwright,
-developers can define and reuse build strategies that build container images for their CI/CD
-pipelines. Any tool that builds images within a container can be supported, such
-as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
-[Cloud Native Buildpacks](https://buildpacks.io/), and [Buildah](https://buildah.io/).
+Shipwright is an extensible framework for building container images on Kubernetes.
 
-## Dependencies
+## Why?
 
-| Dependency                           | Supported versions           |
-| -------------------------------------| ---------------------------- |
-| [Kubernetes](https://kubernetes.io/) | v1.17.\*, v1.18.\*, v1.19.\* |
-| [Tekton](https://tekton.dev)         | v0.21.0                      |
+With Shipwright developers get a simplified approach for building container images, by defining a minimal YAML that does not require
+any previous knowledge of containers or containers tooling. All you need is your source code in git and access to a container registry.
 
-## Build Strategies
-
-The following [Build Strategies](docs/buildstrategies.md) are installed by default:
-
-* [Source-to-Image](docs/buildstrategies.md#source-to-image)
-* [Buildpacks-v3](docs/buildstrategies.md#buildpacks-v3)
-* [Buildah](docs/buildstrategies.md#buildah)
-* [Kaniko](docs/buildstrategies.md#kaniko)
-* [ko](docs/buildstrategies.md#ko)
-
-Users have the option to define their own `BuildStrategy` or `ClusterBuildStrategy` resources and make them available for consumption via the `Build` resource.
+Shipwright can support any tool that can build container images, such as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
+[Cloud Native Buildpacks](https://buildpacks.io/), and [Buildah](https://buildah.io/). See [default strategies](#build-strategies) for more information.
```

Comment:
Like Jordan said, maybe better to avoiding double "can".
```suggestion
Shipwright supports any tool that can build container images, for example [Kaniko](https://github.com/GoogleContainerTools/kaniko),
[Cloud Native Buildpacks](https://buildpacks.io/), or [Buildah](https://buildah.io/). See [default strategies](#build-strategies) for more information.
```

Reply:
done